web/locales/en/messages.ftl

```diff
@@ -324,6 +324,7 @@ off = Off
 on = On
 add-avatar-title = Add an avatar to your profile
 add-avatar-clip = Add an avatar to your profile and say “hello!”
+avatar-clip-title = An avatar to your profile!
```

Comment:
this english looks incomplete

Reply:
I just wanna ask this after opening a PR ! that what should I show as the title if no recording option is available !

Reply:
No, it is not from copy doc